scss/core/_variables.scss

```diff
@@ -1,4 +1,5 @@
 // Variables
+// Variables
```

Comment:
Was this a mistake?

Reply:
yes, and I couldn't figure out what about it changed

Reply:
// Våriables ?